force-app/main/default/triggers/OppTrigger.trigger

```diff
@@ -0,0 +1,11 @@
+trigger OppTrigger on Opportunity (before insert, before update, before delete) {
+    if(Trigger.isBefore){
+        if(Trigger.isUpdate){
+            OppAmountValidation.opportunityAmountHandler(Trigger.new);
+            SetPrimaryContactToOpp.setPrimaryContactOnAccToOpp(Trigger.new);
+        }
+        if(Trigger.isDelete){
+            OppClosedWonDeletionValidation.oppsToBeDeleted(Trigger.old);
+        }
+        }
+        }
```

Comment:
good but work on your formatting. All your curly braces stacked like that makes it hard to read.

Reply:
Sure. I'll make note of this.